stable/prometheus/README.md

```diff
@@ -97,6 +97,7 @@ Parameter | Description | Default
 `alertmanager.image.repository` | alertmanager container image repository | `prom/alertmanager`
 `alertmanager.image.tag` | alertmanager container image tag | `v0.13.0`
 `alertmanager.image.pullPolicy` | alertmanager container image pull policy | `IfNotPresent`
+`alertmanager.imagePullSecrets` | alertmanager container image pull Secrets | `IfNotPresent`
```

Comment:
The default parameter looks like a copy&paste error.

Reply:
changed to `[]` as the default values.